File: Java/Caavo/app/src/main/java/com/dewaara/caavo/Remote/IMenuRequest.java
package com.dewaara.caavo.Remote;


import com.dewaara.caavo.Model.Item;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Url;

public interface IMenuRequest {

    @GET
    Call<List<Item>> getMenuList(@Url String url);
}

File: Kotlin/Caavo/app/src/main/java/com/dewaara/caavo/Remote/IMenuRequest.kt
package com.dewaara.caavo.Remote

import com.dewaara.caavo.Model.Item
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Url

interface IMenuRequest {
    @GET
    fun getMenuList(@Url url: String?): Call<List<Item?>?>
}